lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.html

```diff
@@ -90,7 +90,7 @@
                      class="adf-metadata-grouped-properties-container">
                     <mat-expansion-panel *ngIf="showGroup(group) || editable"
                                          [attr.data-automation-id]="'adf-metadata-group-' + group.title"
-                                         [expanded]="canExpandTheCard(group) || !displayDefaultProperties && first">
+                                         [expanded]="canExpandTheCard(group.title) || !displayDefaultProperties && first">
```

Comment:
Here similar case like above mentioned by @swapnil-verma-gl. Can we correct it too as we touch this place?

Reply:
closing this as discussed in previous comment